src/Symfony/Component/Config/Builder/ClassBuilder.php

```diff
@@ -35,6 +37,7 @@ class ClassBuilder
     public function __construct(
         private string $namespace,
         string $name,
+        private ?NodeInterface $node = null,
```

Comment:
do we have a case where we pass `null` or omit it ? Or should we make it required ?